app/components/meetings/sessions/session-list.vue

```diff
@@ -38,42 +38,83 @@
 
 <script>
 
-import   Api, {mapObjectId}from '../api.js'
-import { dateTimeFilter  } from '../filters.js'
+import   Api           , { mapObjectId }from '../api.js'
+import { dateTimeFilter }               from '../filters.js'
+import   moment                         from 'moment'
 
 export default {
   name       : 'SessionsList',
   props      : { 
                   route:       { type: Object, required: false },
                   tokenReader: { type: Function, required: false }
                 },
+  methods    : { getUrl, meetingQuery,conferenceQuery, sessionNumber, isInProgress, now },
+  computed   : { now },
   filters    : { dateTimeFilter },
-  created, 
-  data,
-  methods: { getUrl} 
+  created, data
 }
 
 function data(){
   return { 
     sessions: [],
-    meetings: []
+    meetings: [], 
+    api: undefined
   }
 }
 
-function getUrl(id) {
-  return `/meetings/${encodeURIComponent(this.route.params.meeting)}/sessions/${encodeURIComponent(id)}`
+function getUrl({ _id, meetings }) {
+  const { symbol } = meetings[0]
+
+  return `/meetings/${encodeURIComponent(symbol)}/sessions/${encodeURIComponent(_id)}`
 }
 
 async function created(){
   this.api = new Api(this.tokenReader);
 
-  let q = { normalizedSymbol: this.route.params.meeting.toUpperCase() };
-  const meetings = await this.api.queryMeetings({q, f: { _id:1, normalizedSymbol:1 }});
+  const isMeeting = await this.meetingQuery();
+  const q         = isMeeting? isMeeting: await this.conferenceQuery();
+
+  this.sessions  = await this.api.querySessions({ q, s: { date:-1 } });
+}
+
+async function meetingQuery(){
+  const normalizedSymbol = this.route?.params?.meeting? this.route?.params?.meeting.toUpperCase() : ''
+
+  if(!normalizedSymbol) return ''
+
+  const q        = { normalizedSymbol };
+
+  this.meetings = await this.api.queryMeetings({q, f: { _id:1, normalizedSymbol:1 }});
+
+  return { meetingIds : { $in: this.meetings.map(m=>mapObjectId(m._id)) }, date: { $lte: { $date: this.now.add(24, 'hours') } } };
+}
+
+async function conferenceQuery(){
+  const { code } = this.route.params
+
+  if(!code) throw new Error('No meeting id or conference code route param for session list')
+
+  const q                              = { 'code': code };
+  const { MajorEventIDs: meetingIds }  = await this.api.getConference(code);
+
+  return { 'conference.symbol': code, meetingIds : { $in: meetingIds.map(m=>mapObjectId(m)) }, date: { $lte: { $date: this.now.add(24, 'hours') } } };
```

Comment:
conference.symbol seems useless as you pass meetingIds
```suggestion
  return { meetingIds : { $in: meetingIds.map(m=>mapObjectId(m)) }, date: { $lte: { $date: this.now.add(24, 'hours') } } };
```

Reply:
Actually, 'conference.symbol': code alone should suffice but does not work.

Reply:
Actually, I'm more on the side that 'conference.symbol' should not be usable as filter field. 
Allowing any fields to use as filter is bad because not all fields has index an it make create a database performance issue. we should have a list of `queryable` fields 

in fact `conference` can be populated dynamically at load time and never be in the db as symbol is mutable